src/signals/incident-management/containers/DefaultTextsAdmin/components/DefaultTextsForm/index.test.js

```diff
@@ -1,10 +1,155 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { withAppContext } from 'test/utils';
 
-// import DefaultTextsForm from 'index';
+import categories from 'utils/__tests__/fixtures/categories.json';
+
+import DefaultTextsForm from './index';
 
 describe('<DefaultTextsForm />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+  let props;
+
+  beforeEach(() => {
+    props = {
+      defaultTexts: [{
+        title: 'title 1',
+        text: 'text 1',
+      }, {
+        title: 'title 2',
+        text: 'text 2',
+      }, {
+        title: 'title 3',
+        text: 'text 3',
+      }],
+      subCategories: categories.sub,
+      categoryUrl: 'https://acc.api.data.amsterdam.nl/signals/v1/public/terms/categories/afval/sub_categories/asbest-accu',
+      state: 'o',
+
+      onSubmitTexts: jest.fn(),
+      onOrderDefaultTexts: jest.fn(),
+      onSaveDefaultTextsItem: jest.fn(),
+    };
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should render form correctly', () => {
+    const fields = [...Array(10).keys()];
+    const { queryByTestId } = render(
+      withAppContext(<DefaultTextsForm {...props} />)
+    );
+
+    expect(queryByTestId('defaultTextFormForm')).not.toBeNull();
+
+    fields.forEach(i => {
+      expect(queryByTestId(`title${i}`)).not.toBeNull();
+      expect(queryByTestId(`text${i}`)).not.toBeNull();
+    });
+
+    expect(queryByTestId('defaultTextFormItemButton0Up')).toBeDisabled();
```

Comment:
I'm pretty sure you can group a lot of these assertions by using a query selector, like so:
```jsx
container.querySelectorAll('button:nth-of-type(n+2)').forEach(button => {
  expect(button).toBeDisabled();
});
```
See https://alistapart.com/article/quantity-queries-for-css/ for reference

Reply:
I have created an alternative